### DISCUSSION TITLE
Frequency of emailing

### DISCUSSION THREAD

**Discussion by JonathanTaylor9** [emailjs-com/help#14] [Q&A]
Is there a limit on frequency of sending emails?  

In testing I'm sending regular emails - if I trigger them from my front end in quick succession (a handful of seconds between submits) then the second email doesn't arrive at the 'Email To' recipient - bizarrely, it does get sent to the 'Cc' recipient though, and if I look in the outbox I can see that the email was addressed to the correct Email To and Cc recipients, it just never arrived in the Email To inbox.  If I wait a few seconds longer between submits, this issue doesn't occur.

**Comment by xr0master** [comment]
Yes, as stated in the documentation, no more than 1 request per second.
This limit **protects our clients** from abuse, and our servers from heavy peak loads.
